releasenotes/notes/features-88f5ec5ec6b5c380.yaml

```diff
@@ -0,0 +1,18 @@
+# Each section from every release note are combined when the
+# CHANGELOG.rst is rendered. So the text needs to be worded so that
+# it does not depend on any information only available in another
+# section. This may mean repeating some details, but each section
+# must be readable independently of the other.
+#
+# Each section note must be formatted as reStructuredText.
+---
+upgrade:
+  - |
+    List upgrade notes here, or remove this section.
+    Upgrade notes should be rare: only list known/potential breaking changes,
+    or major behaviorial changes that require user action before the upgrade.
+    Notes here must include steps that users can follow to 1. know if they're
+    affected and 2. handle the change gracefully on their end.
+features:
+  - |
+    Enabling the option to strip the arguments of the commands collected by the process agent on Windows.
```

Comment:
```suggestion
    Enabling the option to strip the arguments of the commands collected by the process Agent on Windows.
```

Reply:
Thank you @maycmlee , I just updated this :)